distributionviewer/core/static/css/distributionviewer.styl

```diff
@@ -72,9 +71,17 @@ a {
 
   a {
     color: $normal-text;
-    font-size: 14px;
-    font-weight: bold;
+    font-size: 12px;
     text-decoration: none;
+    line-height: 2.5;
+    display: block;
+    border-radius: 0 4px 4px 0;
+    padding: 0 10px 0 0;
```

Comment:
I notice `.filters` also has a `padding-right`. Do the links need one also? I might be missing something.